diff --git a/node_modules/tinacms/dist/index.js b/node_modules/tinacms/dist/index.js
index 8f6829e..d09a92b 100644
--- a/node_modules/tinacms/dist/index.js
+++ b/node_modules/tinacms/dist/index.js
@@ -36,7 +36,8 @@ import * as DropdownMenuPrimitive from "@radix-ui/react-dropdown-menu";
 import * as ToolbarPrimitive from "@radix-ui/react-toolbar";
 import * as SeparatorPrimitive from "@radix-ui/react-separator";
 import * as TooltipPrimitive from "@radix-ui/react-tooltip";
-import colorString, { get as get$6, to as to$1 } from "color-string";
+import colorString from "color-string";
+const { get: get$6, to: to$1 } = colorString;
 import { createForm, FORM_ERROR, getIn } from "final-form";
 import arrayMutators from "final-form-arrays";
 import setFieldData from "final-form-set-field-data";
